### PR TITLE
README translation

### PR DESCRIPTION
**Pull request by innabelaya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,11 @@ It contains base realizations of blocks that are abstracted from the design. Few
 ## Usage
 
 You can use any way you are familiar with to include this library into your project.
-E.g. you can use `bem-tools` or `ENB`.
+E.g. you can use [bem-tools](http://bem.info/tools/bem/bem-tools/) or `ENB`. For this add name, version and path of the library to the configuration file of selected assembler.
```
Comment (tadatuta) [inline README.md]: path **to**?

File: README.md [hunk 2]
```diff
@@ -8,7 +8,11 @@ It contains base realizations of blocks that are abstracted from the design. Few
...
+
+If you use [project-stub](http://ru.bem.info/tutorials/project-stub/) as a base of your project, you do not need to include `bem-components`. It is added to the project by default.
```
Comment (tadatuta) [inline README.md]: remove **ru** in url

Comment (tadatuta) [inline README.md]: you do not need to include `bem-components` **manually**

File: README.md [hunk 3]
```diff
@@ -58,20 +62,43 @@ E.g. you can use `bem-tools` or `ENB`.
 
 ### Technologies
 
-* [JS](https://bem.info/technology/i-bem/current/i-bem-js/)
+* [i-bem.js](https://bem.info/technology/i-bem/current/i-bem-js/)
 * [YModules](https://bem.info/tools/bem/modules/)
 * [BEMHTML](https://bem.info/technology/bemhtml/current/reference/)
-* BH
+* [BH](https://github.com/bem/bh/blob/master/README.md)
```
Comment (tadatuta) [inline README.md]: it's already approved in testing, please ask @dab for url in production

Comment (innabelaya) [inline README.md]: URL to BH description on bem.info will be changed. As soon as the URL will be corrected, I will update this README.

**Pull request closed without merging**